File: atcoder/abc399a_hamming_distance.nim
# Vicfred
# https://atcoder.jp/contests/abc399/tasks/abc399_a
# implementation
import strutils

let n = stdin.readLine.parseInt
let s = stdin.readLine
let t = stdin.readLine

var ans : int
for i in 0..<n:
  if s[i] != t[i]:
    ans += 1
echo ans

File: atcoder/abc399a_hamming_distance.cpp
// Vicfred
// https://atcoder.jp/contests/abc399/tasks/abc399_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int n;
  string s, t;
  cin >> n;
  cin >> s;
  cin >> t;
  int ans = 0;
  for(int i = 0; i < n; ++i) {
    if(s[i] != t[i]) {
      ans += 1;
    }
  }
  cout << ans << endl;
  return 0;
}
